src/components/DevelopmentGuide/V3MigrationGuide.mdx

```diff
@@ -0,0 +1,155 @@
+
+- The `NewsletterSignup` and `SocialMediaLinks` components have been removed.
+  They will be added back in a later version.
+- `SimpleGrid` gap values have been updated to include `"grid"` in the prop
+  value name. For example, `gap=”grid.xs”`.
+- The `Link` component now supports app framework's routing link components. For
+  example, `Link` component from Next.js can be used through the `as` prop. The
+  previous pattern where the framework's `Link` component was used as a parent
+  component is no longer supported.
+
+<Source
+  code={`
+// Assuming that Next.js 13+ is being used
+
+// Do This
+import Link as NextLink from "next/link";
+import { Link } from "@nypl/design-system-react-components";
+
+<Link href="/about" as={NextLink}>
+  About
+</Link>
+
+// Don't Do This
+import Link from "next/link";
+import { Link as DSLink } from "@nypl/design-system-react-components";
+
+<Link href="/about" passHref>
+  <DSLink>About</DSLink>
+</Link>
+`} language="jsx" />
+
+- The `TagSet` `"filter"` variant has been updated to improve filtering logic in
+  consuming apps.
+  - Only for the `"filter"` variant, the `tagSetData` prop now
+    expects an array of objects with the following required properties:
+    `{ label: string, id: string }`. Any additional property can now also be
+    added to the object.
+  - The `onClick` function now returns the entire data object of the clicked
+    tag instead of the tag's "label".
+
+### Chakra UI
+
+- It is recommended to use `cookieStorageManagerSSR` instead of
+  `cookieStorageManager` for color mode server-side rendering (SSR). The returned
+  value should still used in the `DSProvider`'s `colorModeManager` prop.
+- The `FocusLock` component was previously imported from the separate Chakra
+  `@chakra-ui/focus-lock` package. In Chakra v2+, `FocusLock` can now be
+  imported from the main npm Chakra package. This is now included as an export
+  from the DS.
+
+<Source
+  code={`
+    import { FocusLock } from "@nypl/design-system-react-components";
+  `}
+  language="jsx"
+/>
+
+## Additional Testing Needs
+
+- Due to some JS features and how we test, consuming apps need to add the
+  following to their Jest setup file. They are needed for the following
+  components and hooks: `Tabs`, `Slider`, and `useNYPLBreakpoints`.
+  - We are looking into making updates so that consuming apps do not have to
+    include these test helpers in future releases.
+
+<Source
+  code={`
+// Added in Jest setup file as defined in jest.config.js
+
+// Related to the useNYPLBreakpoints hook which is used in: ButtonGroup,
+// FeedbackBox, Modal, MultiSelectGroup, and NewsletterSignup.
+import { MatchMedia } from "@nypl/design-system-react-components";
+
+new MatchMedia();
+
+// The scrollIntoView function is necessary for the Tabs component.
+window.HTMLElement.prototype.scrollIntoView = jest.fn()
+
+// Related to the Slider component
+class ResizeObserver {
+observe() {}
+unobserve() {}
+disconnect() {
+}
```

Comment:
Nit: add two spaces to the lines of code within the class brackets.

Reply:
I tried but it doesn't work! I may need to break this up.